domains/certificates/Shared/DataContext/ApplicationDbContext.cs

```diff
@@ -47,8 +42,6 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
     }
 
     public DbSet<CertificateIssuingContract> Contracts { get; set; }
-    public DbSet<ProductionCertificate> ProductionCertificates { get; set; }
```

Comment:
We dont need these to hold state anymore?

Reply:
No we decided to drop them